<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Find &amp;what:</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="caseCheckBox">
         <property name="text">
          <string>Match &amp;case</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="backwardCheckBox">
         <property name="text">
          <string>Search &amp;backward</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="findButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Find</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>FindDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>FindDialog</receiver>
   <slot>enableFindButton(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>FindDialog</receiver>
   <slot>findClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enableFindButton(QString)</slot>
  <slot>findClicked()</slot>
 </slots>
</ui>
